firefox-ios/Client/Frontend/Browser/TabScrollController.swift

```diff
@@ -598,8 +598,6 @@ private extension TabScrollController {
             } else {
                 overKeyboardContainerOffset = overKeyboardOffset
             }
-            overKeyboardContainer?.updateAlphaForSubviews(alpha)
-            overKeyboardContainer?.superview?.layoutIfNeeded()
```

Comment:
I've removed these two lines to restore the changes that were added in the minimal toolbar animation [PR](https://github.com/mozilla-mobile/firefox-ios/pull/27852/files#diff-54d15d839db358f095a0e341b4593b9267791e2c9975fb32c4eae17b15ef5584R601-R602), because they caused a bug.